addons/flink-kubernetes-operator/metadata.yaml

```
@@ -0,0 +1,21 @@
+name: flink-kubernetes-operator
+version: release-0.1.0
```

```suggestion
version: 0.1.0
```
please set with a `smever` style version, otherwise the helm index cannot be handled correctly.